web/libs/editor/src/tags/control/ClassificationBase.js

```diff
@@ -49,6 +49,15 @@ const ClassificationBase = types
         }
         return self.annotation.results.find((r) => r.from_name === self);
       },
+
+      /**
+       * That name historically was used for regions, but now it's used for the classifications as well.
+       * Let's say "Region" here means just an area on the screen.
+       * So that it's an element through which we can get the bbox For an area where classification takes place.
+       */
+      getRegionElement() {
```

Comment:
while `getRegionElement()` in Result is a mimicry to Region's one, here it's a different method with a little different semantic. Maybe `getRenderedElement()`?

Reply:
In that case I would have to change `DOMWhatcher` that uses this method. I mean, I can do it, but how many places it would affect...